ofborg/src/tasks/eval/nixpkgs.rs

```diff
@@ -26,6 +26,12 @@ use uuid::Uuid;
 
 static MAINTAINER_REVIEW_MAX_CHANGED_PATHS: usize = 64;
 
+const TITLE_LABELS: [(&str, &str); 3] = [
+    ("darwin", "6.topic: darwin"),
+    ("macos", "6.topic: macos"),
```

Comment:
Nit, but we don't have a macos label :P

```suggestion
    ("macos", "6.topic: darwin"),
```

Reply:
Fixed!  I also alphabetised the list because that seemed like a sensible thing to do.